tests/security/swtpm/tpm_tools_test.pm

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Update TrouSerS and tpm-tools to the latest version
+#          Make sure your setup has a TPM 1.2 device attached,
+#          both hardware or software devices should be fine
+#          We need only test this feature on aarch64 platform,
+#          However, based on bsc#1193350, test it on x86_64 as
+#          well
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#103644, tc#1769832
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils qw(zypper_call systemctl);
+use Utils::Architectures;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Version check
+    my $pkg_list = {'tpm-tools' => '1.3.9.2', trousers => '0.3.15'};
+    foreach my $pkg (keys %$pkg_list) {
+        zypper_call("in $pkg");
+        my $current_ver = script_output("rpm -q --qf '%{version}\n' $pkg");
+        record_info("Current $pkg version is $current_ver, target version is $pkg_list->{$pkg}");
```

Comment:
@rfan1 
Looks like you miss one parameter in the record_info, right?

Reply:
Because I cannot see any content in these blocks.
https://openqa.suse.de/tests/7898034#step/tpm_tools_test/12
https://openqa.suse.de/tests/7898034#step/tpm_tools_test/24

Reply:
Should we rename the pm file to such as 'tpm_selftest.pm" since test case does not test tpm_tools.

Reply:
which one?

Reply:
For example:
record_info('openssl version', "Version of Current openssl package: $current_ver");

Reply:
I catch your points, Only title is defined, but no results there. let me fix it....thanks much